pkg/envoy/lds/inmesh_test.go

```diff
@@ -285,6 +287,35 @@ func TestBuildInboundHTTPFilterChain(t *testing.T) {
 			expectedFilterNames: []string{envoy.L4GlobalRateLimitFilterName, envoy.HTTPConnectionManagerFilterName},
 			expectError:         false,
 		},
+		{
+			name:           "inbound HTTP filter chain with global HTTP rate limiting enabled",
+			permissiveMode: true,
+			trafficMatch: &trafficpolicy.TrafficMatch{
+				Name:                "inbound_ns1/svc1_90_http",
+				DestinationPort:     90,
+				DestinationProtocol: "http",
+				ServerNames:         []string{"svc1.ns1.svc.cluster.local"},
+				RateLimit: &policyv1alpha1.RateLimitSpec{
+					Global: &policyv1alpha1.GlobalRateLimitSpec{
+						HTTP: &policyv1alpha1.HTTPGlobalRateLimitSpec{
+							RateLimitService: policyv1alpha1.RateLimitServiceSpec{
+								Host: "foo.bar",
+								Port: 8080,
```

Comment:
do these two values need to be checked?

Reply:
Not required in this test, but I'll add a new test to ensure the OSM config -> Envoy config translation happens correctly.